render/npm.render

```diff
@@ -415,7 +415,7 @@ app ingest {
       severity 0
       1: u64 cgroup
       2: u64 cgroup_parent
-      3: u8 name[64]
+      3: u8 name[256]
```

Comment:
this message is sent from kernel collector to reducer (in `buffered_poller.cc`).

unfortunately, the protocol currently does not support forwards and backwards compatibility (we implemented most of the mechanism but had not put everything together..).

what we have been doing so far when messages change (and we should consider doing here), is to create a new message with a new RPC ID that has a new format. Then, the old handler calls the new handler after translating the message. For example `pid_info_create_deprecated` and `pid_info_create`.

Reply:
done (I think - are the rpc ids scoped, or do I need to avoid number collisions throughout the entire file?)